From 4955fcd106f765689af7aed72e37634e15fb4f36 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 31 Jul 2014 16:51:58 +0800
Subject: [PATCH 351/602] ENGR00242365 Update BUILD_ID for KK4.4.3_2.0.0-Beta
 release

Update BUILD_ID as "2.0.0-beta-rc3"
Update BUILD_NUMBER as "20140815"

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 4 ++--
 arm2_6sl/build_id.mk       | 4 ++--
 evk_6sl/build_id.mk        | 4 ++--
 hdmidongle_6dq/build_id.mk | 4 ++--
 imx5x/build_id.mk          | 4 ++--
 sabreauto_6q/build_id.mk   | 4 ++--
 sabresd_6dq/build_id.mk    | 4 ++--
 sabresd_6sx/build_id.mk    | 4 ++--
 8 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index 8bb40da..08bd82b 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index 8bb40da..08bd82b 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index 8bb40da..08bd82b 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index 8bb40da..08bd82b 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index 8bb40da..08bd82b 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index 8bb40da..08bd82b 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index 8bb40da..08bd82b 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
diff --git a/sabresd_6sx/build_id.mk b/sabresd_6sx/build_id.mk
index 8bb40da..08bd82b 100644
--- a/sabresd_6sx/build_id.mk
+++ b/sabresd_6sx/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc2
-export BUILD_NUMBER=20140806
+export BUILD_ID=2.0.0-beta-rc3
+export BUILD_NUMBER=20140815
-- 
1.8.0

